### DISCUSSION TITLE
Why does a resource need to have a permalink to be editable?

### DISCUSSION THREAD

**Discussion by MtDalPizzol** [statamic-rad-pack/runway#205] [Help] [answered]
Hi folks. I'm trying out Runway and it seems promising. But my resources doesn't show `View` and `Edit` buttons on the dropdown menu.

![image](https://user-images.githubusercontent.com/3843774/215173798-f7b02545-88b7-461f-af7a-dd2c928f11f3.png)

I did a little code diving and find out that the resource need to be [editable and to have a permalink](https://github.com/duncanmcclean/runway/blob/4accec2d9399366f5a4f132c1fe624db16eb56d2/resources/js/components/Listing/ListingView.vue#L181). I do have permission to edit the resource and `editable` is true on the JSON received from the server.

![image](https://user-images.githubusercontent.com/3843774/215174517-4c9a0ebc-6f87-43a0-9417-adbe3e353757.png)

However, there's some resources that don't need to have a permalink. I just need to manage them in the CP. I even tried adding a `route` to the resource in the config file, but even with this I'm still getting null as the permalink.

```php
    'resources' => [
        \App\Models\SubscriptionPlans::class => [
            'name' => 'Subscription Plans',
            'blueprint' => 'collections.subscription_plans.subscription_plan',
            'route' => '/planos/{id}'
        ],
```

So... Not sure how to proceed here. Don't know if there's a good reason for that, if it's a bug or if I did something wrong.

**Comment by duncanmcclean** [comment, marked as answer]
It's a bug - will be fixed shortly!

**Reply by MtDalPizzol** [reply]
Well... that's some pretty quick feedback right there! Thanks! Will stick with the package for now, see how far it can take me!

**Reply by duncanmcclean** [reply]
Fixed in both [v2.6.6](https://github.com/duncanmcclean/runway/releases/tag/v2.6.6) and [v3.0.1](https://github.com/duncanmcclean/runway/releases/tag/v3.0.1)

I don't usually tag two versions for a bug fix but didn't want to force you to update to v3 and Statamic 3.4 just after I tagged it 😅

**Reply by MtDalPizzol** [reply]
Awesome! Thanks! I'll try this today... I'm probably upgrading to v3 anyway, since Statamic 3.4 is looking hot!
